diff --git a/src/afl-fuzz-init.c b/src/afl-fuzz-init.c
index af6e6d4c..e4492a50 100644
--- a/src/afl-fuzz-init.c
+++ b/src/afl-fuzz-init.c
@@ -2412,7 +2412,7 @@ void setup_stdio_file(afl_state_t *afl) {
   unlink(afl->fsrv.out_file);                              /* Ignore errors */
 
   afl->fsrv.out_fd =
-      open(afl->fsrv.out_file, O_RDWR | O_CREAT | O_EXCL, DEFAULT_PERMISSION);
+      open(afl->fsrv.out_file, O_RDWR | O_CREAT | O_EXCL, 0666);
 
   if (afl->fsrv.out_fd < 0) {
 
@@ -3262,4 +3262,3 @@ void save_cmdline(afl_state_t *afl, u32 argc, char **argv) {
   *buf = 0;
 
 }
-
diff --git a/src/afl-sharedmem.c b/src/afl-sharedmem.c
index 1dea83f9..e09cfac2 100644
--- a/src/afl-sharedmem.c
+++ b/src/afl-sharedmem.c
@@ -180,7 +180,7 @@ u8 *afl_shm_init(sharedmem_t *shm, size_t map_size,
 
       shm->g_shm_fd =
           shm_create_largepage(shm->g_shm_file_path, shmflags, i,
-                               SHM_LARGEPAGE_ALLOC_DEFAULT, DEFAULT_PERMISSION);
+                               SHM_LARGEPAGE_ALLOC_DEFAULT, 0666);
 
     }
 
@@ -192,7 +192,7 @@ u8 *afl_shm_init(sharedmem_t *shm, size_t map_size,
   if (shm->g_shm_fd == -1) {
 
     shm->g_shm_fd =
-        shm_open(shm->g_shm_file_path, shmflags | O_CREAT, DEFAULT_PERMISSION);
+        shm_open(shm->g_shm_file_path, shmflags | O_CREAT, 0666);
 
   }
 
@@ -364,4 +364,3 @@ u8 *afl_shm_init(sharedmem_t *shm, size_t map_size,
   return shm->map;
 
 }
-
